shared/src/test/scala/mlscript/DiffTests.scala

```diff
@@ -446,41 +446,76 @@ class DiffTests extends org.scalatest.funsuite.AnyFunSuite with org.scalatest.Pa
                   }
                   // Execute code.
                   if (!mode.noExecution) {
+                    if (mode.showRepl) {
+                      output(s"┌ Block at ${file.last}:${blockLineNum}")
+                    }
+                    // Execute the prelude code.
                     prelude match {
-                      case Nil => ()
-                      case lines => host.execute(lines mkString " ")
+                      case Nil => {
+                        if (mode.showRepl) {
+                          output(s"├── No prelude")
+                          if (queries.isEmpty)
+                            output(s"└── No queries")
+                        }
+                      }
+                      case lines => {
+                        val preludeReply = host.execute(lines mkString " ")
+                        if (mode.showRepl) {
+                          output(s"├─┬ Prelude")
+                          output(s"│ ├── Code")
+                          lines.iterator.foreach { line => output(s"│ │   $line") }
+                          // Display successful results in multiple lines.
+                          // Display other results in single line.
+                          preludeReply match {
+                            case ReplHost.Result(content, intermediate) =>
+                              intermediate.foreach { value =>
```

Comment:
`value` is not used here. Seems like a bug.

Reply:
Thank you so much! It’s a bug caused by copy'n'paste.